tests/testthat/test-preserve-names.R

```diff
@@ -0,0 +1,147 @@
+x <- c(C = "3", B = "2", A = "1")
```

Comment:
I think it would make more sense to put each test in the file corresponding to its function. And these tests should all be self-contained; i.e. it's better to redefine `x` inside of each of them so they are easy to run interactively.

Reply:
Will do.